java/arcs/android/devtools/StoreMessage.kt

```diff
@@ -0,0 +1,163 @@
+package arcs.android.devtools
+
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_MESSAGE
+import arcs.core.common.Referencable
+import arcs.core.crdt.CrdtData
+import arcs.core.crdt.CrdtOperation
+import arcs.core.crdt.CrdtSet
+import arcs.core.crdt.CrdtSingleton
+import arcs.core.crdt.VersionMap
+import arcs.core.data.RawEntity
+import arcs.core.data.util.ReferencablePrimitive
+import arcs.core.storage.ProxyMessage
+import arcs.core.util.JsonValue
+
+/**
+ * An implementation of [DevToolsMessage] to inform DevTools that a [Store] received a
+ * [CrdtOperation].
+ */
+class StoreMessage(
```

Comment:
Nice job!

Something that I would like you to be aware of / keep an eye on longer term:

In the medium term as we resurrect Web-bits, we should aim to have the protocol defined in an implementation independent way and be able to use the same Developer Tools in both Android and Web embeddings of Arcs.

So while it's ok to have the messages defined in code for the short term (including all of Q4 I think), we should avoid putting in things that are implementation specific already.